scripts/addresses/1-tmp-assets-collateral.json

```diff
@@ -61,7 +61,8 @@
     "sfrxETH": "0x4c891fCa6319d492866672E3D2AfdAAA5bDcfF67",
     "apxETH": "0x05ffDaAA2aF48e1De1CE34d633db018a28e3B3F5",
     "sUSDe": "0x35081Ca24319835e5f759163F7e75eaB753e0b7E",
-    "pyUSD": "0xa5cde4fB1132daF8f4a0D3140859271208d944E9"
+    "pyUSD": "0xa5cde4fB1132daF8f4a0D3140859271208d944E9",
+    "stUSD": "0xb6aD2903fAC2db628624DeE61e217F50Ad54638E"
```

Comment:
i don't see anything deployed here https://etherscan.io/address/0xb6aD2903fAC2db628624DeE61e217F50Ad54638E

Reply:
Yep reverting, it was deployed on a local mainnet fork.